crates/bevy_reflect/derive/src/ident.rs

```diff
@@ -0,0 +1,47 @@
+use proc_macro2::{Ident, Span};
+use syn::Member;
+
+/// Returns the "reflected" ident for a given string.
+///
+/// # Example
+///
+/// ```
+/// # use proc_macro2::Ident;
+/// # // We can't import this method because of its visibility.
+/// # fn get_reflect_ident(name: &str) -> Ident {
+/// #     let reflected = format!("Reflect{name}");
+/// #     Ident::new(&reflected, proc_macro2::Span::call_site())
+/// # }
+/// let reflected: Ident = get_reflect_ident("Hash");
+/// assert_eq!("ReflectHash", reflected.to_string());
+/// ```
+pub(crate) fn get_reflect_ident(name: &str) -> Ident {
+    let reflected = format!("Reflect{name}");
+    Ident::new(&reflected, Span::call_site())
+}
+
+/// Returns a [`Member`] made of `ident` or `index` if `ident` is None.
```

Comment:
Nit: While we're here, we could fix some of these doc comments
```suggestion
/// Returns a [`Member`] made of `ident` or `index` if `ident` is `None`.
```